setup.py

```diff
@@ -79,6 +80,40 @@ def _patched_copy_file(src_file, dest_file, hide_listing=True):
 skbuild.setuptools_wrap._copy_file = _patched_copy_file
 
 
+class BuildPyCmd(_skbuild_build_py):
+    def copy_file(self, src, dst, preserve_mode=True):
+        _patched_copy_file(src, dst, preserve_mode=preserve_mode)
+        return (dst, 1)
+
+
+class InstallCmd(_skbuild_install):
```

Comment:
Will it work for `python setup.py develop` too? or only for `python setup.py intell` ?

Reply:
It works with both, and since the test was added to the test suite, CI is checking it for install in conda-packages workflow, and for decelop in generate-coverage and build-with-os-llvm workflows.